acnportal/acnsim/events/tests/test_event_queue.py

```diff
@@ -54,7 +54,6 @@ def test_get_last_timestamp(self):
         self.assertEqual(5, self.events.get_last_timestamp())
         self.events.add_event(Event(8))
         self.assertEqual(8, self.events.get_last_timestamp())
-        curr_events = self.events.get_current_events(3)
```

Comment:
What is the propose of the second

` self.assertEqual(8, self.events.get_last_timestamp())`

 if we remove 
`curr_events = self.events.get_current_events(3)`

I'm actually not sure what we were testing in the first place.

Reply:
It looks like we were testing that the last timestamp is not affected by calling get_current_events on an earlier timestamp. It doesn't seem necessary, but if it was a bug that caused us to add such a condition, maybe it's safer to keep the test unaltered for now. I'll add a comment detailing what the test does.